pkg/certmaker/templates/leaf-template.json

```diff
@@ -0,0 +1,28 @@
+{
+    "subject": {
+        "country": [
+            "US"
+        ],
+        "organization": [
+            "Sigstore"
+        ],
+        "organizationalUnit": [
+            "Fulcio Leaf CA"
+        ],
+        "commonName": "https://fulcio.com"
+    },
+    "issuer": {
+        "commonName": "https://fulcio.com"
+    },
+    "notBefore": "2024-01-01T00:00:00Z",
+    "notAfter": "2034-01-01T00:00:00Z",
+    "basicConstraints": {
```

Comment:
Should we allow these values to be set in the template? Given that the certificates should conform to the cert profile, which is strict on basic constraints, key usage and extended key usage, can the template just be for subject, issuer, not before/after?

Then this is simplified to a single template for leaf/intermediate/root.

Reply:
that makes a lot of sense, but as you mention below, provide an option to set the cert life (e.g. flag / env var).

Reply:
okidoki, i include templating and a embed...so now templates are included w/ the binary as well as a positional arg for common name.